streamz/dask.py

```diff
@@ -103,7 +104,14 @@ def update(self, x, who=None, metadata=None):
         client = default_client()
 
         self._retain_refs(metadata)
-        future = yield client.scatter(x, asynchronous=True)
+        # We need to make sure that x is treated as it is by dask
+        # However, client.scatter works internally different for
+        # lists and dicts. So we always use a dict here to be sure
+        # we know the format exactly. The key will be taken as the
+        # dask identifier of the data.
+        tokenized_x = type(x).__name__ + "-" + tokenize(x)
```

Comment:
I would have done this as an fstring

Reply:
We *just* moved to py>=3.6 :)

Reply:
Nice :-) I just copied this from dask, but you are very correct!